config/database.yml

```diff
@@ -18,7 +18,7 @@ development:
 # Do not set this db to the same as development or production database.
 test:
   <<: *default
-  database: tmp/test.sqlite3
+  database: db/test.sqlite3
```

Comment:
What does this change do? Is it important? :-)

Reply:
It is only the folder where test.sqlite3 is saved. The folder is not important but I change it to db so every .sqlite3 file is in the same folder ;)